content/documentation/architecture.adoc

```diff
@@ -0,0 +1,116 @@
+---
+title: "Architecture"
+date: 2018-09-03T14:39:48-05:00
+draft: false
+---
+
+= Kiali architecture
+:imagesdir: /images/documentation/architecture
+
+Kiali is composed of two components: a backend running in the container
+application platform, and a user frontend. However, in order to be useful,
```

Comment:
I would remove "in order to be useful" , no?
Maybe just: "Moreover, Kiali depends on ..."

Reply:
Ok, I changed it using your suggestion.